### PR TITLE
Run test with different python versions

### PR DESCRIPTION
**Pull request by shagren**
Tests will be fired for 3.6, 3.7 and 3.8 python.
Not sure is it required.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,4 +5,5 @@ flake8-isort==4.0.0
 mypy==0.782
 mypy-extensions==0.4.3
 pytest==6.0.1
-pytest-cov==2.10.1
\ No newline at end of file
+pytest-cov==2.10.1
+dataclasses==0.6; python_version<"3.7"
```
Comment (shagren) [inline requirements-dev.txt:9]: Dataclasses used only in tests then requirements in this file

File: tests/plugins/device.py [hunk 2]
```diff
@@ -228,7 +228,10 @@ def device_get_imu_sample(
             assert self._opened is True
             if not self._cameras_started:
                 return Result.Failed.value, None
-            return Result.Success.value, (36.6, (0.1, 9.8, 0.005), time.time_ns(), (0.1, 0.2, 0.3), time.time_ns())
+            return (
+                Result.Success.value,
+                (36.6, (0.1, 9.8, 0.005), int(time.time() * 1000), (0.1, 0.2, 0.3), int(time.time() * 1000)),
+            )
```
Comment (shagren) [inline tests/plugins/device.py:234]: time.time_ns() appeared in 3.7.

Comment (etiennedub) [reply tests/plugins/device.py:234]: I might be wrong, but is it `time.time()` in second and it should be multiply by 1000^2 to be in ns?

Comment (lpasselin) [reply tests/plugins/device.py]: s*1e9 == ns

Comment (lpasselin) [reply tests/plugins/device.py]: Actually the imu timestamp is in us so it should be 1e6
https://microsoft.github.io/Azure-Kinect-Sensor-SDK/master/structk4a__imu__sample__t.html

**Pull request merged**